backend/arm64/cfg_selection.ml

```diff
@@ -173,6 +173,52 @@ class selector =
       (* Other operations are regular *)
       | _ -> super#select_operation op args dbg ~label_after
 
+    method! emit_stores env dbg data regs_addr =
+      let offset = ref (-Arch.size_int) in
```

Comment:
I think it deserves a comment.
While the reason for the override
can be inferred from the code, it
is probably useful to have a couple
of lines explaining why it is needed.

Reply:
Added comments.